server/command.go

```diff
@@ -145,6 +147,22 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 	ctx, cancel := context.WithTimeout(context.Background(), commandTimeout)
 	defer cancel()
 
+	defer func() {
+		if r := recover(); r != nil {
+			p.client.Log.Warn("Recovered from a panic",
+				"Command", args.Command,
+				"UserId", args.UserId,
+				"error", r,
+				"stack", string(debug.Stack()))
+			err = model.NewAppError("ExecuteCommand", "plugin_gitlab.command.execute_command.panic", nil, "", http.StatusInternalServerError)
+			res = &model.CommandResponse{}
+			p.postCommandResponse(args, "An unexpected error occurred. Please try again later.")
+			if *p.client.Configuration.GetConfig().ServiceSettings.EnableDeveloper {
+				p.postCommandResponse(args, r.(string))
```

Comment:
2/5 that it's helpful to also print the stack (`debug.Stack()`)

Reply:
Thanks! Updated the code.